src/modules/postCreation/components/NoteCreation.tsx

```diff
@@ -17,6 +18,9 @@ import PostCreationButtons from './PostCreationButtons';
 import PageTitle from '../../../lib/components/Pages/PageTitle';
 import { CreatePostRequestType } from '../../../lib/utilities/API/types';
 import { createPost } from '../../../lib/utilities/API/api';
+import BorderBottom from '../../../lib/components/Border';
+import UrlInputModal from '../../../lib/components/Editor/CustomModules/UrlInputModal';
```

Comment:
```suggestion
import UrlInputModal from '../../../lib/components/Editor/CustomModules/UrlInputModal';
```
I suggest using named export to avoid such a situation where we can rename the original name of the component, what do u think ?

severity: ***recommendation***

Reply:
I agree with the suggestion, therefore files exporting was adjusted in accordance with "export convention"